hazelcast/src/hazelcast/client/spi/impl/ClientPartitionServiceImpl.cpp

```diff
@@ -48,16 +47,21 @@ namespace hazelcast {
                 void ClientPartitionServiceImpl::RefreshTaskCallback::onFailure(
                         const boost::shared_ptr<exception::IException> &t) {
                     if (partitionService.client.getLifecycleService().isRunning()) {
-                        partitionService.logger.warning() << "Error while fetching cluster partition table! Cause:" << t;
+                        partitionService.logger.warning() << "Error while fetching cluster partition table! Cause:"
+                                                          << t;
                     }
                 }
 
                 ClientPartitionServiceImpl::ClientPartitionServiceImpl(ClientContext &client) : client(client),
```

Comment:
Can you please fix the formatting here by starting  `:client(client),` from one line below. This is impossible to read.